export/wiki/items/status.py

```diff
@@ -1,10 +1,35 @@
-def convert_status_effect(entry):
+from typing import Optional, Tuple, Union
+
+from automate.hierarchy_exporter import ExportModel, Field
+from ue.properties import FloatProperty
+
+
+class StatEffectData(ExportModel):
+    value: FloatProperty = Field(..., title="Stat value to gain")
+    useItemQuality: bool = Field(False, title="Whether item quality is used")
+    descriptionIndex: int = Field(..., title="")
+    pctOf: Optional[str] = Field(None, title="")
+    pctAbsRange: Optional[Tuple[FloatProperty, FloatProperty]] = Field(None, title="")
+    setValue: bool = Field(False, title="")
+    setAddValue: bool = Field(False, title="")
+    forceUseOnDino: bool = Field(False, title="Whether can be used on dinos")
+    allowWhenFull: bool = Field(True, title="Whether allowed when stat is full")
+    qualityMult: float = Field(1.0, title="Item quality effect (addition)")
+    duration: Union[float, FloatProperty] = Field(0, title="Effect duration")
```

Comment:
Use `FloatLike` instead of your own union. Could probably use it in place of all of your floats and remove the casts.